data/src/main/java/com/linkedin/data/schema/resolver/ClasspathResourceDataSchemaResolver.java

```diff
@@ -33,61 +34,106 @@
  *
  * @author Min Chen
  */
-public class ClasspathResourceDataSchemaResolver extends DefaultDataSchemaResolver
```

Comment:
This class no longer extends `DefaultDataSchemaResolver`. Are you sure you're not removing any default functionality it had before?

Reply:
Since this delegates to the SingleFormat resolver (inner class) which extends the Default resolver, there shouldn't be any regression.

Reply:
Ah, that's right. It's interesting how the relationship of the multi-format resolvers with the default resolver logic is more "has-a" than "is-a". Reminds me of CS class...